mlir/lib/Catalyst/Transforms/InlineNestedModules.cpp

```diff
@@ -272,6 +319,8 @@ struct InlineNestedModule : public RewritePattern {
 
         return success();
     }
+
+    llvm::SmallSet<StringRef, 8> _externalFuncDeclNames;
```

Comment:
Can this be in the same location as the other datastructure? Throws me off that they are split 😅